lib/domain/rss1_feed.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:core';
+
+import 'package:webfeed/domain/dublin_core/dublin_core.dart';
+import 'package:webfeed/domain/rss1_item.dart';
+import 'package:webfeed/util/helpers.dart';
+import 'package:xml/xml.dart';
+
+enum UpdatePeriod {
+  Hourly,
+  Daily,
+  Weekly,
+  Monthly,
+  Yearly,
+}
+
+class Rss1Feed {
```

Comment:
instead of naming this `Rss1Feed`, what if we call this `RdfFeed`

https://validator.w3.org/feed/docs/rss1.html